app/modules/editor/contentObject/views/editorPageView.js

```diff
@@ -47,15 +47,18 @@ define(function(require){
       this._onScroll = _.bind(_.throttle(this.onScroll, 400), this);
     },
 
-    render: function() {
-      Origin.editor.data.content.fetch({
-        success: () => {
-          var returnVal = EditorOriginView.prototype.render.apply(this, arguments);
-          this.addArticleViews();
-          return returnVal;
-        },
-        error: () => Origin.Notify.alert({ type: 'error', text: Origin.l10n.t('app.errorfetchingdata') })
-      });
+    render: async function() {
+      try {
+        await Promise.all([
+          Origin.editor.data.content.fetch(),
+        ]);
```

Comment:
We don't need the `Promise.all` here, as we're just doing one fetch:

```suggestion
        await Origin.editor.data.content.fetch();
```

Otherwise, looks good :)